tools/build.py

```diff
@@ -88,6 +89,10 @@ def devhelp(help):
         parser.print_help()
         sys.exit(0)
 
+    if arguments.jerry_debugger == 'ON' and arguments.jerry_libc == 'ON':
+        print('JerryScript Debugger need the standard libc. Please use the \'--jerry-libc=off\' for it.')
```

Comment:
I would prefer to check this somewhere in the CMake source with an error message. I think it would be better, especially if you build with the cmake only and don't want to use the script.